iceoryx_examples/waitset/README.md

```diff
@@ -124,21 +124,23 @@ are stored inside of the **EventInfo** and can be acquired by the user.
 
 This example consists of 5 use cases.
```

Comment:
6 use cases

Reply:
it seems this is the result of an party unsuccessful merge/rebase. Could maybe be fixed by some rebase black magic

Reply:
It is, and it is quite likely that we will accidentally desync even more with merges and rebases when working on two (slightly) different branches.

Reply:
Corrected by changing the last commit.